apps/admin/oauth2-proxy/oauth2-proxy.yaml

```diff
@@ -72,6 +75,8 @@ spec:
     ingress:
       annotations:
         traefik.ingress.kubernetes.io/router.tls: "true"
+      hosts:
+        - oauth-proxy.demo.platform.hmcts.net
```

Comment:
Remove these again. They will get picked up from the cluster specific file when deployed via flux